api/notd/collection_processor.py

```diff
@@ -74,6 +74,8 @@ async def retrieve_collection(self, address: str) -> RetrievedCollection:
         if contractMetadataUri is not None:
             if contractMetadataUri.startswith('ipfs://'):
                 contractMetadataUri = contractMetadataUri.replace('ipfs://', 'https://ipfs.io/ipfs/')
+            if "address" in contractMetadataUri:
```

Comment:
ah you should try not to use format like this - its a safety risk. this is super unlikely but imagine someone had like `https://myurl.com/{DB_PASSWORD}` and we used format it would populate the var in (if it existed in local scope) and then we send it to their server